api/models.py

```diff
@@ -18,42 +18,6 @@ def __str__(self) -> str:
         return self.english
 
 
-class TranslationUserAccuracyCounter(models.Model):
```

Comment:
Doesn't this require a migration?

Reply:
i think it works without it because it is unused, though existent in db, will migrate